shotgun_api3/shotgun.py

```diff
@@ -97,6 +97,10 @@ class AuthenticationFault(Fault):
     """Exception when the server side reports an error related to authentication"""
     pass
 
+class MultiFactorAuthenticationFault(Fault):
+    """Exception when the server side reports an error related to missing 2FA credentials"""
```

Comment:
Changed to "Missing...Fault", The "auth_token" is ignored (even if given) when 2FA is not turned on.